src/App.js

```diff
@@ -16,7 +16,13 @@ class App extends Component {
       popUpGenre: null,
       searching: '',
       filteredGames: [],
-      filterOptions: []
+      filterOptions: [
+        false, true,
+        false, true, false, false,
+        false, false, false, false,
+        false, false, false, false,
+        false, false, false, false, false
+      ]
     }
```

Comment:
1st line has card games as true and second line has 8-13 age range as true hardcoded for testing